src/Console/Command/DispatchCommand.php

```diff
@@ -207,52 +207,80 @@ private function dispatchFiles(Package $package)
             ->config('user.email', static::GITHUB_EMAIL)
         ;
 
-        foreach ($projectConfig['branches'] as $branch => $branchConfig) {
-            $this->io->section('Files for '.$branch);
+        $branches = array_reverse($projectConfig['branches']);
+
+        $previousBranch = null;
+        $previousDevKit = null;
+        while (($branchConfig = current($branches))) {
+            // We have to fetch all branches on each step in case a PR is submitted.
+            $remoteBranches = array_map(function ($branch) {
+                return $branch['name'];
+            }, $this->githubClient->repos()->branches(static::GITHUB_GROUP, $repositoryName));
+
+            $currentBranch = key($branches);
+            $currentDevKit = $currentBranch.'-dev-kit';
+            next($branches);
```

Comment:
Putting this right in the middle of the `while` statement feels a bit weird… why not line 215 ?